.github/workflows/release.yml

```diff
@@ -0,0 +1,146 @@
+        run: mvn install
+
+      - name: Build sbt plugin
+        working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin
+        run: sbt compile publishLocal --debug
+
+      - name: Build sbt test
+        working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin
+        run: sbt scripted
+
+      - name: Build gradle example-server --warning-mode all
+        run: ./gradlew -p plugins/gradle/example-server test --warning-mode all
+
+      - name: Build gradle example-client
+        run: ./gradlew -p plugins/gradle/example-client test --warning-mode all
+
+      - name: Build gradle example-client-kotlin
+        run: ./gradlew -p plugins/gradle/example-client-kotlin build --warning-mode all
+
+      - name: Build maven example-server
+        working-directory: plugins/maven/example-server
+        run: mvn package
+
+      - name: Build maven example-client
+        working-directory: plugins/maven/example-client
+        run: mvn package
+
+      - name: Install gpg secret key
+        run: cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
+
+      #
+      # RELEASE STEPS
+      #
+
+      - name: Release library
+        run: |
+          ./gradlew publish \
+              -Dorg.gradle.internal.publish.checksums.insecure=true \
+              -Pgradle.publish.key=${{ secrets.GRADLE_PUBLISH_KEY}} \
+              -Pgradle.publish.secret=${{ secrets.GRADLE_PUBLISH_SECRET }}
+        env:
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+          SIGNING_KEY: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
+          SIGNING_PASSWORD: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
+          RELEASE_VERSION: ${{ github.event.inputs.release_version }}
+
+      - name: Release Gradle plugin
+        run: |
+          ./gradlew \
+              -p plugins/gradle/graphql-java-codegen-gradle-plugin \
+              publishPlugins \
+              -Dorg.gradle.internal.publish.checksums.insecure=true \
+              -Pgradle.publish.key=${{ secrets.GRADLE_PUBLISH_KEY}} \
+              -Pgradle.publish.secret=${{ secrets.GRADLE_PUBLISH_SECRET }}
+        env:
+          SIGNING_KEY: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
+          SIGNING_PASSWORD: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
+          RELEASE_VERSION: ${{ github.event.inputs.release_version }}
+
+      - name: Release Maven plugin
+        working-directory: plugins/maven/graphql-java-codegen-maven-plugin
+        run: |
+          mvn \
+            --no-transfer-progress \
+            --batch-mode \
+            -Dgpg.passphrase=${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }} \
+            release:clean release:prepare release:perform
+
+      - name: Release SBT plugin
+        working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin
+        run: sbt release with-defaults default-tag-exists-answer k
+
+
+      - name: Commit and push the new version
```

Comment:
I guess the script will remove SNAPSHOT from all files before publishing, and the next version to be published will be submitted after publishing, but I wonder if gradle and Maven will automatically change to the next version after release? Because SBT will. So, What does `new version` in `Commit and push the new version` mean?

Reply:
maven will automatically increment the version **only for the plugin**, but not the example projects.
gradle **will not** automatically increment the version at all (plugin + example projects)
library version will also **not** be incremented automatically.

So I will update versions using the script I wrote in this PR before commit/push.

Reply:
Everything looks good, but I'm afraid there might be something wrong with `with-defaults` in Non interactive environment. And OSS account has been sent to your Google email. If there is any problem in the test, please inform me to change the SBT task at any time.

Reply:
I didn’t receive any email yet.. did you send it to 92bogdan@gmail.com?

Reply:
yes

Reply:
I used outlook mailbox, garbage can to see?